rpc/params/crossmodel.go

```diff
@@ -523,6 +570,37 @@ type RegisterRemoteRelationArg struct {
 	// RemoteEndpoint contains info about the endpoint in the remote model.
 	RemoteEndpoint RemoteEndpoint `json:"remote-endpoint"`
 
+	// OfferUUID is the UUID of the offer.
+	OfferUUID string `json:"offer-uuid"`
+
+	// LocalEndpointName is the name of the endpoint in the local model.
+	LocalEndpointName string `json:"local-endpoint-name"`
+
+	// ConsumeVersion is incremented each time a new consumer
+	// proxy is created for an offer.
+	ConsumeVersion int `json:"consume-version,omitempty"`
+
+	// Macaroons are used for authentication.
+	Macaroons macaroon.Slice `json:"macaroons,omitempty"`
+
+	// BakeryVersion is the version of the bakery used to mint macaroons.
+	BakeryVersion bakery.Version `json:"bakery-version,omitempty"`
+}
+
+// RegisterRemoteRelationArg holds attributes used to register a remote relation.
+type RegisterRemoteRelationArgV2 struct {
```

Comment:
Just inline embed RegisterRemoteRelationArg and have the RemoteSpace attr in addition.

Reply:
done!